lib/components/viewers/related-stops-panel.js

```diff
@@ -100,7 +100,7 @@ class RelatedStopsPanel extends Component {
                     }
                   />
                 </div>
-                <div className="pattern-list">
+                <div className="pattern-list" role="list">
```

Comment:
Looks like using `<ul>` [may be recommended](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#:~:text=available%20and%20supported.-,Avoid%20using%3A,-application) over using `role="list"`

Reply:
Good catch thanks!